### PR TITLE
Create Curl

### PR DESCRIPTION
**Pull request by ghost**
#406 
#405 
#402 #403

### CODE REVIEW

File: Curl
Comment (JustDontsnitchonme) [inline Curl:1]: #407

**Review by ghost — COMMENTED** [review]
#407

**Comment by samruddhikhandale** [comment]
Looks like this was created by accident, closing. If needed, feel free to reopen with more details, thank you!

**Review by ghost — COMMENTED** [review]
[CI](https://github.com/cloudflare/cloudflare-python/actions/workflows/ci.yml)

**Review by ghost — COMMENTED** [review]
@bamurtaugh

**Pull request closed without merging**